react-app/src/components/ButtonsModal/index.js

```diff
@@ -1,48 +1,83 @@
-import React, { PureComponent } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 
 import BaseModal from 'components/BaseModal';
 
 import styles from './index.module.scss';
 
-export default class ButtonsModal extends PureComponent {
-  static propTypes = {
-    /**
-     * Whenever the modal is visible or not
-     */
-    visible: PropTypes.bool.isRequired,
-
-    /**
-     * The message that the modal will show
-     */
-    message: PropTypes.string.isRequired,
-
-    /**
-     * The primary button (a Button) that the element will show
-     */
-    primaryButton: PropTypes.element.isRequired,
-
-    /**
-     * The (optional) secondary button (a Button) that the element will show
-     */
-    secondaryButton: PropTypes.element,
-  };
-
-  static defaultProps = { secondaryButton: null }
-
-  render() {
-    return (
-      <BaseModal visible={this.props.visible}>
-        <div className={styles.container}>
+const ButtonsModal = props => {
+  const {
+    visible,
+    message,
+    primaryButton,
+    secondaryButton,
+  } = props;
+
+  function _getButtons() {
+    if (!secondaryButton) {
+      return (
+        <div className={styles.buttonsContainer}>
           <div>
-            {this.props.message}
+            {primaryButton}
           </div>
-          <div className={styles.buttonsContainer}>
-            {this.props.secondaryButton}
-            {this.props.primaryButton}
+        </div>
+      );
+    } else {
+      return (
+        <div className={styles.buttonsContainer}>
+          <div />
```

Comment:
Is this empty div here on purpose?

Reply:
I use it (as the one at the end of the container) to push away the buttons to the center of the modal, since the container has `justify-content: space-around`.

It doesn't seem to me to be the best way of doing this, but I left it because it worked and I don't need to add some extra styles. Any suggestions?

Reply:
I have re-implemented this just adding lateral padding to the button(s).